### PR TITLE
[Horizon] Assignment Details

### PR DESCRIPTION
**Pull request by szabinst**
Adds a basic assignment details screen with mock description.

### CODE REVIEW

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>Build failed, skipping coverage check</td>
    </tr>
  </tbody>
</table>



<details><summary>:x: Swift lint</summary>


    
    ----- xcbeautify -----
    Version: 2.11.0
    ----------------------
    
    ❌ /Users/vagrant/git/Horizon/Horizon/Sources/Features/AssignmentDetails/Data/HAssignment.swift:78:1: Line Length Violation: Line should be 200 characters or less; currently it has 775 characters (line_length)
    /Users/vagrant/git/Horizon/Horizon/Sources/Features/AssignmentDetails/Data/HAssignment.swift:80:1: error: Line Length Violation: Line should be 200 characters or less; currently it has 307 characters (line_length)
    
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 51020ea5cfd447aa330b2350d3aca71df89d152f
</p>

File: Horizon/Horizon/Sources/Common/View/LearningObjectHeaderView.swift [hunk 1]
```diff
@@ -19,7 +19,14 @@
 import Core
 import SwiftUI
 
-struct ContentDetailsView: View {
+struct LearningObjectHeaderView: View {
```
Comment (Ahmed-Naguib93) [inline Horizon/Horizon/Sources/Common/View/LearningObjectHeaderView.swift:22]: I think renaming it from **LearningObjectHeaderView** to **AssignmentHeaderView** would be more meaningful.

**Review by Ahmed-Naguib93 — CHANGES_REQUESTED** [review]

File: Horizon/Horizon/Sources/Common/View/LearningObjectHeaderView.swift (hunk 1 above)
Comment (szabinst) [reply Horizon/Horizon/Sources/Common/View/LearningObjectHeaderView.swift:22]: We'll use it at other places not just for assignments, hence the name. And Learning Object is the new term used in Horizon.

**Review by Ahmed-Naguib93 — APPROVED** [review]

**Comment by inst-danger** [comment]
<details><summary>Horizon Build QR Code:</summary>

</details>

**Review by Ahmed-Naguib93 — APPROVED** [review]

**Pull request merged**